src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx

```diff
@@ -3,141 +3,114 @@
 
 interface Props {
-  workPackage: WorkPackage;
   users: User[];
-  setters: any;
+  control: any;
+  errors: any;
 }
 
-const WorkPackageEditDetails: React.FC<Props> = ({ workPackage, users, setters }) => {
-  const editDetailsInputBuilder = (
-    title: string,
-    type: string,
-    defaultValue: any,
-    updateState: ((val: string) => void) | null,
-    prefix = '',
-    suffix = '',
-    placeholder = '',
-    readOnly = false
-  ) => {
-    return (
-      <Form.Group>
-        <Form.Label>{title}</Form.Label>
-        <InputGroup>
-          {prefix ? <InputGroup.Text>{prefix}</InputGroup.Text> : <></>}
-          <Form.Control
-            required
-            type={type}
-            min={0}
-            defaultValue={defaultValue}
-            placeholder={placeholder}
-            readOnly={readOnly}
-            onChange={(e) => {
-              if (updateState !== null) {
-                updateState(e.target.value);
-              }
-            }}
-          />
-          {suffix ? <InputGroup.Text>{suffix}</InputGroup.Text> : <></>}
-        </InputGroup>
-      </Form.Group>
-    );
-  };
-
-  const statuses = Object.values(WbsElementStatus).filter((status) => status !== workPackage.status);
-
-  const transformStatus = (status: string) => {
-    switch (status) {
-      case 'ACTIVE':
-        return WbsElementStatus.Active;
-      case 'INACTIVE':
-        return WbsElementStatus.Inactive;
-      default:
-        return WbsElementStatus.Complete;
-    }
-  };
+const statuses = Object.values(WbsElementStatus);
 
+const WorkPackageEditDetails: React.FC<Props> = ({ users, control, errors }) => {
   const statusSelect = (
-    <Form.Control as="select" onChange={(e) => setters.setStatus(transformStatus(e.target.value))} custom>
-      <option key={0} value={workPackage.status}>
-        {workPackage.status}
-      </option>
-      {statuses.map((status, index) => (
-        <option key={index + 1} value={status}>
-          {status}
-        </option>
-      ))}
-    </Form.Control>
-  );
-
-  const buildUsersSelect = (title: string, defaultUser: User | undefined, updateUser: (val: number) => void) => {
-    let otherUsers = users;
-    if (defaultUser !== undefined) {
-      otherUsers = users.filter((user) => user.userId !== defaultUser.userId);
-    }
-    return (
-      <Form.Group>
-        <Form.Label>{title}</Form.Label>
-        <Form.Control as="select" data-testid={title} onChange={(e) => updateUser(parseInt(e.target.value))} custom>
-          {defaultUser === undefined ? (
-            <option key={-1} value={-1}>
-              {emDashPipe('')}
-            </option>
-          ) : (
-            <option key={defaultUser.userId} value={defaultUser.userId}>
-              {fullNamePipe(defaultUser)}
-            </option>
-          )}
-          {otherUsers.map((user) => (
-            <option key={user.userId} value={user.userId}>
-              {fullNamePipe(user)}
-            </option>
+    <Controller
```

Comment:
i think this should be in line after the start date, the gap between work package details and the name is too big for it to be up here